specification/datasets/cost_and_usage/columns/effectivecost.md

```diff
@@ -1,6 +1,6 @@
 # Effective Cost
 
-Effective Cost represents the [*amortized*](#glossary:amortization) cost of the [*charge*](#glossary:charge) after applying all reduced rates, discounts, and the applicable portion of relevant, prepaid purchases (one-time or recurring) that covered this *charge*. The *amortized* portion included should be proportional to the [Pricing Quantity](#pricingquantity) and the time granularity of the data. Since amortization breaks down and spreads the cost of a prepaid purchase, to subsequent eligible *charges*, the Effective Cost of the original prepaid *charge* is set to 0. Effective Cost does not mix or "blend" costs across multiple *charges* of the same [*service*](#glossary:service). This cost is denominated in the [Billing Currency](#billingcurrency). The Effective Cost is commonly utilized to track and analyze spending trends.
+Effective Cost represents the sum of [*amortized*](#glossary:amortization) upfront fees, amortized recurring fees, and Non-prepaid costs of the [*charge*](#glossary:charge) after applying all reduced rates, discounts, and the applicable portion of applicable pre-payment (one-time or recurring) that covered this charge. The amortized portion included should be proportional to the [Pricing Quantity](#pricingquantity) and the time granularity of the data. Since amortization breaks down and spreads the cost of a prepaid purchase, to subsequent eligible *charges*, the Effective Cost of the original prepaid *charge* is set to 0. Effective Cost does not mix or "blend" costs across multiple charges of the same [*service*](#glossary:service). This cost is denominated in the [Billing Currency](#billingcurrency). The Effective Cost is commonly utilized to track and analyze spending trends from a usage perspective.
```

Comment:
Would the *use* of an upfront commitment/drawdown purchase, paid on first hour of its term, equally distributed over all months, and available for variably consumption be captured  in EffectiveCost?